src/index.tsx

```diff
@@ -20,13 +20,14 @@
 
 // window.__deps = dependencies
 
-import React, { Component } from "react"
-import { render } from "react-dom"
-import { Grommet, ResponsiveContext } from "grommet"
-import { theme } from "./theme"
-import { IntlProvider } from 'react-intl';
-import { BrowserRouter as Router, Switch, Route } from "react-router-dom"
-import { Dashboard, HelloWorld } from "./components"
+import React, { Component } from 'react'
+import { render } from 'react-dom'
+import { Grommet, ResponsiveContext } from 'grommet'
+import { theme } from './theme'
+import { IntlProvider } from 'react-intl'
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
+import { Dashboard, HelloWorld } from './components'
+import { CustomerListView } from './views/customer/app'
```

Comment:
Let's rename the file `./view/customer/app` to `./view/customer/CustomerListView`